packages/preset-create-react-app/package.json

```diff
@@ -24,15 +24,18 @@
   "dependencies": {
     "@storybook/node-logger": "^5.2.0",
     "@types/webpack": "^4.41.2",
+    "pnp-webpack-plugin": "^1.6.4",
     "react-docgen-typescript-loader": "^3.6.0",
-    "semver": "^7.1.1"
+    "semver": "^7.1.3"
   },
   "devDependencies": {
-    "@types/semver": "^6.2.0",
+    "@types/semver": "^7.1.0",
     "typescript": "^3.7.5"
   },
   "peerDependencies": {
     "@storybook/react": ">=5.2",
-    "react-scripts": ">=3.0.0"
+    "node-sass": "*",
+    "react-scripts": ">=3.0.0",
+    "webpack": "^3.0.0 || ^4.0.0"
```

Comment:
Again, this is not a dependency of this preset IMO. The `react-scripts` compatibility range would take care of any issues around versions :)

Reply:
Same as `node-sass`